docusaurus.config.js

```diff
@@ -99,7 +107,7 @@ module.exports = {
           ],
         },
       ],
-      copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+      copyright: `Copyright © OpenJS Foundation and Electron contributors.`,
```

Comment:
I removed the year because the generated JSON had it hardcoded somehow instead of doing interpolation and thus it will be kept out of date.